test/book/specialized/fang-fourier-monomial-bases/gap.jlcon

```diff
@@ -0,0 +1,2 @@
+L:= SimpleLieAlgebra("A", 4, Rationals);; 
+V:= HighestWeightModule(L, [1,3,2,1]);; 
```

Comment:
Ugh, this is GAP code. Will see how we can resolve this in the book (we don't want that tested as Julia code, it can't work)

Reply:
same here. this is just a code snipped used to produce timings, so this should neither have output nor get tested. ping @gfourier